graphql/e2e/custom_logic/custom_logic_test.go

```diff
@@ -2708,7 +2708,7 @@ func TestCustomDQL(t *testing.T) {
 				screen_name: "minhaj"
 				followers: 10
 			  }
-			  timestamp: "2021-02-23"
+			  timestamp: "2022-02-23"
```

Comment:
Not sure if we need to update testing data.

Reply:
+1 ^

Reply:
^ +1